spi/common/catalog-spi/src/main/java/org/eclipse/edc/catalog/spi/Distribution.java

```diff
@@ -0,0 +1,85 @@
+/*
+ *  Copyright (c) 2023 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.catalog.spi;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+
+import java.util.Objects;
+
+@JsonDeserialize(builder = Distribution.Builder.class)
+public class Distribution {
+
+    private String format;
+    private DataService dataService;
```

Comment:
In the DCAT spec the Distribution contains a reference (id) to the DataService. For internal handling having the DataService object seemed easier. When transforming back from DCAT JSON-LD (where only the DataService id is present), a DataService with just an ID would have to be constructed, meaning null values for other DataService fields would have to be permitted, even though it would technically be an invalid DataService. Therefore I decided to go with this way, but this could definitely be further discussed or refactored to have just the ID there. Wdyt?